### DISCUSSION TITLE
PDF Export Probleme

### DISCUSSION THREAD

**Discussion by bothropsasper2** [ifl-geovis/hin-und-weg#4] [Bugs & Probleme]
Hallo,

ich versuche derzeit "Visualisierungen", die mit "hin&weg" erstellt wurden, als PDFs zu exportieren. Diese Funktion wurde u.a. im Video "Teil 3: Analyse- und Visualisierungsfunktionen" (8.12.2021) erklärt. Allerdings lässt sich der PDF-Export für mich nicht wie im Video reproduzieren: 
- Die "Kontrollelemente"-Zeile ist einerseits niemals ausgeblendet (im Video heißt es, diese würde beim PDF-Export ausgeblendet)
- das größte Problem: Die eigentliche Grafik ist niemals richtig zentriert (erkennbar im Screenshot) und damit nicht für Veröffentlichungen geeignet. Der Ausschnitt, der exportiert wird, ist stattdessen abhängig vom "Zoom" beim Export. Aber selbst bei "neutralem" Zoom wird nicht die eingelesene Shapedatei beim PDF-Export in den Mittelpunkt genommen.

Daher wäre meine Frage, was bei der Anwendung von hin&weg beachtet werden muss, damit passable PDF und/oder Bild-Exporte gelingen, die zentriert sind und keine "Kontrollelemte"-Schaltfläche anzeigen. Ist das mit hin&weg überhaupt möglich und wenn ja wie?

Vielen Dank für Ihre Hilfe.

**Comment by JGescher** [comment]
Hallo,

leider ist das ein bekanntes Problem bei der Verwendung von Leaflet. Wir arbeiten im Moment an einer Lösung, um einen einfachen Export ohne Verschiebungen zu ermöglichen. 
Bis dahin kann über die Funktion "Drucken" eine PDF exportiert werden. Dabei ist es wichtig die Seitengröße so einzustellen, dass die Karte komplett darauf passt, also z.B. A3 Querformat. Je nach Qualitätsansprüchen kann natürlich auch einfach ein Screenshot der Karte gemacht werden, dort kommt es ebenfalls zu keinen Verschiebungen.
Die Kontrollelemente Zeile sollte eingeklappt sein, wenn das im Programm so eingestellt wurde. Sie ist bei der Kartenansicht aber weiterhin sichtbar. 

Ich hoffe, dass hilft weiter, ansonsten bräuchte ich Informationen zum Betriebssystem, etc. um dem genauen Problem auf den Grund gehen zu können.

Viele Grüße

Jonathan Gescher/IfL
